### PR TITLE
adds shipctl

### PR DESCRIPTION
**Pull request by kanthchandra**
fixes https://github.com/dry-dock/u14/issues/60

### CODE REVIEW

File: shipctl [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -e
+service_function=$1
+shift
+arguments=$@
+
+if [ "$service_function" == "" ] || [ "$service_function" == "" ]; then
+  echo "Usage: shippable SERVICE COMMAND"
```
Comment (rageshkrishna) [inline shipctl]: Oops, just noticed this... `shipctl`, not `shippable`.

**Comment by kanthchandra** [comment]
I'm closing this pull request for now. I'll reopen this tomorrow.

File: shipctl [hunk 2]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -e
+service_function=$1
+shift
+arguments=$@
+
+if [ "$service_function" == "" ] || [ "$service_function" == "" ]; then
+  echo "Usage: shipctl SERVICE COMMAND"
```
Comment (rageshkrishna) [inline shipctl]: I think we should change `SERVICE COMMAND` to `COMMAND ARGUMENTS`.

File: shipctl [hunk 3]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -e
+service_function=$1
```
Comment (rageshkrishna) [inline shipctl]: Let's rename this to `shipctl_command`

File: shipctl [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+elif [ "$service_function" == "retry" ]; then
+  shippable_retry $arguments
+elif [ "$service_function" == "decrypt" ]; then
+  shippable_decrypt $arguments
+elif [ "$service_function" == "service" ]; then
+  shippable_service $arguments
+elif [ "$service_function" == "replace" ]; then
+  shippable_replace $arguments
+else
+  echo "The action that are trying to perform is not supported"
```
Comment (rageshkrishna) [inline shipctl]: Let's make this `The command $shipctl_command is not supported`

**Pull request merged**